libs/common/src/sql-db/init.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -m
+
+/opt/mssql/bin/sqlservr &
+
+# Wait for SQL Server to start
+sleep 30s
```

Comment:
Is there not a better way to wait for the server to start? 😅

Reply:
Yeah, good call. Updated.